nifi-registry-core/nifi-registry-framework/pom.xml

```diff
@@ -294,12 +294,17 @@
         <dependency>
             <groupId>com.h2database</groupId>
             <artifactId>h2</artifactId>
-            <version>1.4.196</version>
+            <version>1.4.199</version>
```

Comment:
I noticed that this version is newer than what Flyway knows about so we get this warning on start up:
`2019-08-19 12:19:17,793 WARN [main] o.f.core.internal.database.base.Database Flyway upgrade recommended: H2 1.4.199 is newer than this version of Flyway and support has not been tested.
`
We can't upgrade Flyway yet because we need spring-boot to upgrade their Flyway support first.

It looks like 1.4.197 is the last version that doesn't produce this warning, we were on 196 before. Even though everything seems to work fine with 199, I don't love everyone getting this warning in their logs every time they start, so I'm leaning towards going with 197.